src/components/gui/gui.jsx

```diff
@@ -119,6 +119,6 @@ GUIComponent.propTypes = {
     vm: PropTypes.instanceOf(VM).isRequired
 };
 GUIComponent.defaultProps = {
-    basePath: '/'
+    basePath: './'
```

Comment:
Is this still needed after removing `basePath` & `BASE_PATH` in all those other places?